advisories/github-reviewed/2025/09/GHSA-5m5x-9j46-h678/GHSA-5m5x-9j46-h678.json

```diff
@@ -1,19 +1,14 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-5m5x-9j46-h678",
-  "modified": "2025-09-10T20:41:39Z",
+  "modified": "2025-09-10T20:41:40Z",
   "published": "2025-09-09T18:31:25Z",
   "aliases": [
     "CVE-2025-57665"
   ],
   "summary": "Element Plus Link component (el-link) implements insufficient input validation for the href attribute",
-  "details": "Element Plus Link component (el-link) prior to 2.11.0 implements insufficient input validation for the href attribute, creating a security abstraction gap that obscures URL-based attack vectors. The component passes user-controlled href values directly to underlying anchor elements without protocol validation, URL sanitization, or security headers. This allows attackers to inject malicious URLs using dangerous protocols (javascript:, data:, file:) or redirect users to external malicious sites. While native HTML anchor elements present similar risks, UI component libraries bear additional responsibility for implementing security safeguards and providing clear risk documentation. The vulnerability enables XSS attacks, phishing campaigns, and open redirect exploits affecting applications that use Element Plus Link components with user-controlled or untrusted URL inputs.",
-  "severity": [
-    {
-      "type": "CVSS_V4",
-      "score": "CVSS:4.0/AV:N/AC:L/AT:N/PR:N/UI:N/VC:N/VI:H/VA:N/SC:N/SI:N/SA:N/E:U"
-    }
-  ],
```

Comment:
Would a maintainer be able to revert this change? The web UI refused to let me submit the form with this string as it was originally